syncopy/specest/freqanalysis.py

```diff
@@ -22,22 +22,26 @@
 
 # Local imports
 from .mtmfft import MultiTaperFFT
-from .mtmconvol import MultiTaperFFTConvol
-from .wavelet import get_optimal_wavelet_scales, WaveletTransform
+
+from .wavelet import get_optimal_wavelet_scales
 from .const_def import (
     spectralConversions,
     availableTapers,
     availableWavelets,
     availableMethods
 )
-from .compRoutines import SuperletTransform
+from .compRoutines import (
+    SuperletTransform,
+    WaveletTransform,
+    MultiTaperFFTConvol
+)
 
 __all__ = ["freqanalysis"]
 
 
-@unwrap_cfg
-@unwrap_select
-@detect_parallel_client
+# @unwrap_cfg
+# @unwrap_select
+# @detect_parallel_client
```

Comment:
Why are the decorators deactivated?

Reply:
Ah sorry, during dev I often comment them out to better trace the problems/error.. my bad!